.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+version: 2.1
+jobs:
+  test:
+    working_directory: ~/ci
+    docker:
+      # Primary container image where all steps run.
+      - image: 487920318758.dkr.ecr.us-west-2.amazonaws.com/cljs-web3-ci:latest
+        aws_auth:
+            aws_access_key_id: $AWS_ACCESS_KEY_ID
+            aws_secret_access_key: $AWS_SECRET_ACCESS_KEY
+      # Secondary container image on common network.
+      - image: ipfs/go-ipfs
```

Comment:
This is not related to ipfs, is this image needed?

Reply:
Nope. Removed along with adding a placeholder test